owncloudData/src/main/java/com/owncloud/android/data/ClientManager.kt

```diff
@@ -37,24 +38,32 @@ class ClientManager(
     private val preferencesProvider: SharedPreferencesProvider,
     val context: Context,
     val accountType: String,
+    private val connectionValidator: ConnectionValidator
 ) {
     // This client will maintain cookies across the whole login process.
     private var ownCloudClient: OwnCloudClient? = null
 
+    init {
+        SingleSessionManager.setConnectionValidator(connectionValidator)
+    }
```

Comment:
We inject the client manager via koin and I'm not sure if we can skip this `init` block in some way because it could be redundant. What do you think?